src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1148,6 +1154,8 @@ typedef struct __ClientInfo {
     // Version of the struct
     UINT32 version;
 
+    // ------------------------------- V0 compat ----------------------

```

Comment:
The previous compat lines were correct.  They indicate that everything above the comment line is compatible with that version.  V0 included up until the log metric.  See this commit is where `ClientInfo` was introduced and in the same commit the `CLIENT_INFO_CURRENT_VERSION` was set to `0`.

Reply:
Fixed